### ISSUE TITLE
HyperText doesn't render content in body

### ISSUE THREAD

**Issue by beasteers** [heartexlabs/label-studio#1477]
**Describe the bug**
HyperText doesn't accept content from its body and throws an error if value is not set. This means that any hypertext needs to be set in the task data which is not great, especially if you want to nest some static HTML (like a header or whatever).

I found this release (0.5.0) that said that it was now possible so I'm not sure why it isn't possible anymore.
[release 0.5.0](https://github.com/heartexlabs/label-studio/blob/b482c09a0a4b81ec41e0fe2cd73c84ea3c37bf66/docs/source/blog/release-050.md#misc)

**To Reproduce**
```jsx
// this throws an error: Validation failed on : 
//    OrderedDict([('name', 'video'), ('$', 'asdf')]) is not of type 'array'
(<HyperText name="video">asdf</HyperText>)
// this renders nothing
(<HyperText name="video" value="">asdf</HyperText>)
// this renders properly
(<HyperText name="video" value="asdf"></HyperText>)
// this throws an error, meaning that you can't add static html to the template
(<HyperText name="video" value="<h1>hi</h1>"/>)
```
(parentheses just for syntax highlighting)

**Expected behavior**
These should work equivalently.

```jsx
(<HyperText name="video">asdf</HyperText>)
(<HyperText name="video" value="asdf"></HyperText>)
```
And we should be able to do this in a template:
```jsx
(<HyperText name="video"><h1>hi</h1></HyperText>)
```

**Environment (please complete the following information):**
 - OS: [e.g. iOS] mac osx 10.15.7
 - Label Studio Version: latest as of 9/17/21 (1.2.0 I think?)

**Additional context**
I was trying to reproduce the video timeline segmentation example without having to manually change the task data

```jsx
<View>
  <Header value="Video timeline segmentation via AudioPlus sync trick"/>
  <HyperText name="video">
    <video src='$videoSource' width="100%" muted="true" />
    <img src="" onerror="$=n=>document.querySelector(n);a=$('.ls-editor audio');v=$('video');a.onseeked=()=>{v.currentTime=a.currentTime};a.onplay=()=>v.play();a.onpause=()=>v.pause()" />
  </HyperText>
  <Labels name="tricks" toName="audio" choice="multiple">
    <Label value="Kickflip" background="#1BB500" />
    <Label value="360 Flip" background="#FFA91D" />
    <Label value="Trick" background="#358EF3" />
  </Labels>
  <AudioPlus name="audio" value="$videoSource" speed="false"/>
</View>
```

I'm guessing part of the issue is the validation defined here: https://github.com/heartexlabs/label-studio/blob/316cac9052fe6fca3547041d0c321b7182ea2e99/label_studio/core/utils/schema/label_config_schema.json#L71

But I'm not sure where the logic around whether HyperText should use value vs children is.

**Comment by makseq** [comment]
@beasteers Thank you for your feedback, It's a good point to have body inside of HyperText. We completely rewrote the hypertext engine and this feature has been dropped due to low popularity.

BTW you can try to use this labeling config for the video timeline segmentation:

```
<View>
  <Header value="Watch the video:"/>
  <Video name="video" value="$video_url" sync="audio1"/>
  <AudioPlus name="audio1" value="$video_url" sync="video"/>
  <Labels name="lbl" toName="audio1">
    <Label value="One"/>
    <Label value="Two"/>
  </Labels>
</View>
```

**Comment by beasteers** [comment]
Ah nice! That sync prop is very slick!

Yeah I mean I don't know how hard it would be to implement, but I think it's an good feature to have, especially because of how powerful and flexible it is to be able to sprinkle custom html and js in a template! Basically, it seems like a simple feature to add that unlocks a lot of potential.

**Comment by makseq** [comment]
@beasteers Yes, you are absolutely right, I think we'll bring this feature back.

**Comment by makseq** [comment]
@beasteers Could you check the latest master? I think we've fixed it.

**Comment by hlomzik** [comment]
It was fixed in https://github.com/heartexlabs/label-studio/pull/1551
